daemon/graphdriver/counter.go

```diff
@@ -27,7 +27,7 @@ func (c *RefCounter) Increment(path string) int {
 	c.mu.Lock()
 	m := c.counts[path]
 	if m == nil {
-		m = &minfo{check: true}
+		m = &minfo{}
```

Comment:
Hmmm.., even this is outdated? What am I doing wrong. I pulled in your graph-restore branch locally and looking at it.